package/cloudshell/iac/terraform/models/tf_service.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+
+from cloudshell.api.cloudshell_api import CloudShellAPISession, AttributeNameValue
+
+from cloudshell.iac.terraform.constants import ATTRIBUTE_NAMES
+
+
+class TerraformServiceObject(object):
+    def __init__(self, api: CloudShellAPISession, res_id: str, name: str, cloudshell_model_name: str):
+        self.api = api
+        self.res_id = res_id
+        self.name = name
+        self.cloudshell_model_name = cloudshell_model_name
+        self.attributes = self._set_context_resource_attributes()
+        self._check_uuid()
+
+    def _check_uuid(self):
+        if f"{self.cloudshell_model_name}.{ATTRIBUTE_NAMES.UUID}" not in self.attributes:
```

Comment:
maybe f"{self.cloudshell_model_name}.{ATTRIBUTE_NAMES.UUID}" should be moved to a variable?

Reply:
Not sure it`s needed...what are the benefits? why would we want to create a separation ?
*It be easier/coding-practice-friendly not to create special cases

Reply:
I didn't say class. I said put "f"{self.cloudshell_model_name}.{ATTRIBUTE_NAMES.UUID" into a variable because it is copy pasted like 5 or 6 times in this method.

Reply:
corrected